helm/harbor-config/templates/harborconfiguration-crd.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
We will need to see how we handle this CRD. Having it in templates will likely cause trouble.

Reply:
What'll be the recommended approach to handling CRDs?

Reply:
I would personally prefer this way:
1. Use the builtin CRD folder for helm https://github.com/giantswarm/kyverno-app/tree/main/helm/kyverno/crd
2. Use a job to keep them up to date (helm only does install time apply) https://github.com/giantswarm/kyverno-app/tree/main/helm/kyverno/templates/crd-install

Examples taken from the kyverno app

Reply:
I am fine to do this in a follow up - so we can get this merged quicker. Just create an issue please.

Reply:
Alright, issue created - [here](https://github.com/giantswarm/harbor-config-operator/issues/16)